src/Parser/Template.cs

```diff
@@ -54,6 +57,95 @@ public Template(string template, bool isPositional, List<Literal> literals, List
       Holes = holes.ToArray();
     }
 
+    public string Render(object[] parameters)
```

Comment:
I think it's nice to OR create a new class, OR to create a `partial` class :) The latter only when there's a performance impact